dashboard/server/README.md

```diff
@@ -0,0 +1,26 @@
+# StratoCyberLab Dashboard - Server
+
+## Development
+
+- `pip install -r requirements.txt && pip install -r requirements-dev.txt` to 
+  install dependencies (including development tools)
+- `python app.py` to run the development server (also done in Docker)
```

Comment:
@HappyStoic, this actually won't work because [`init` in app.py](https://github.com/stratosphereips/stratocyberlab/blob/4f360e8100b25403f637cb1d8d87d62edd09e199/dashboard/server/app.py#L23) uses absolute paths. I'd say let's make them configurable through env variables, along with the port. What do you think?

Reply:
True. 

> I'd say let's make them configurable through env variables, along with the port. What do you think?

Yea, that will be better. Also I guess inlucing the ollama and hackerlab connection strings. I will create an issue with this so it does not block this PR